We checked the reads against the publicly available genomes for _Ceratobasidium theobromae_ from Sulawesi, Malaysia, Laos and a _Ceraobasidium_ sp. genome from China and viewed in IGV.

```bash

module load  minimap2/2.21-r1071
module load samtools/1.9

cd $PBS_O_WORKDIR

run minimap2 script example - qsub -v prefix_name=GCA_009078325 minimap.pbs 

minimap2 -d ${prefix_name}.mmi ${prefix_name}.fa
minimap2 -K 2g -t 4 -x map-ont -o $OUTDIR/${prefix_name}.sam -a $WDIR/${prefix_name}.mmi $WDIR/HD-amplicons/fastq/barcode24/combined.fastq.gz

Steps to Process SAM/BAM Files

Define input and output file paths
SAMFILE="$OUTDIR/${prefix_name}.sam"
UNMAPPED_SAM="$OUTDIR/${prefix_name}_unmapped.sam"
MAPPED_SAM="$OUTDIR/${prefix_name}_mapped.sam"
UNMAPPED_BAM="$OUTDIR/${prefix_name}_unmapped.bam"
MAPPED_BAM="$OUTDIR/${prefix_name}_mapped.bam"
UNMAPPED_SORTED_BAM="$OUTDIR/${prefix_name}_unmapped_sorted.bam"
MAPPED_SORTED_BAM="$OUTDIR/${prefix_name}_mapped_sorted.bam"

Extract unmapped and mapped reads
samtools view -f 4 -@ 4 "$SAMFILE" > "$UNMAPPED_SAM"
samtools view -F 4 -@ 4 "$SAMFILE" > "$MAPPED_SAM"

Convert SAM to BAM
samtools view -S -@ 4 -b "$UNMAPPED_SAM" > "$UNMAPPED_BAM"
samtools view -b -T "${prefix_name}.fa" "$MAPPED_SAM" > "$MAPPED_BAM"

Sort and index BAM files
for bam_file in "$UNMAPPED_BAM" "$MAPPED_BAM"; do
    sorted_bam="${bam_file%.bam}_sorted.bam"
    samtools sort -m 7G -o "$sorted_bam" -@ 4 "$bam_file"
    samtools index -b -@ 4 "$sorted_bam"
done
