We checked the reads against the publicly available genomes for _Ceratobasidium theobromae_ from Sulawesi, Malaysia, Laos and a _Ceraobasidium_ sp. genome from China.

``
module load  minimap2/2.21-r1071
module load samtools/1.9
``

Working directory

``
cd $PBS_O_WORKDIR
``

run minimap2 script example - qsub -v prefix_name=GCA_009078325 minimap.pbs 

``
minimap2 -d ${prefix_name}.mmi ${prefix_name}.fa
``

Usage: minimap2 [options] <target.fa>|<target.idx> [query.fa] [...] '-a' outputs in sam format then use samtools view to get unmapped with -f 4
``
minimap2 -K 2g -t 4 -x map-ont -o $OUTDIR/${prefix_name}.sam -a $WDIR/${prefix_name}.mmi $WDIR/HD-amplicons/fastq/barcode24/combined.fastq.gz
``
``
samtools view -f 4 -@ 4 $OUTDIR/${prefix_name}.sam > $OUTDIR/${prefix_name}_unmapped.sam
samtools view -F 4 -@ 4 $OUTDIR/${prefix_name}.sam > $OUTDIR/${prefix_name}_mapped.sam
samtools view -S -@ 4 -b $OUTDIR/${prefix_name}_unmapped.sam > $OUTDIR/${prefix_name}_unmapped.bam
``
The headers are missing from the mapped.sam and so we need to fix with the following.

``
samtools view -b -T ${prefix_name}.fa $OUTDIR/${prefix_name}_mapped.sam > $OUTDIR/${prefix_name}_mapped.bam
#samtools view -S -@ 4 -b $OUTDIR/${prefix_name}_mapped.sam > $OUTDIR/${prefix_name}_mapped.bam
samtools sort -m 7G -o $OUTDIR/${prefix_name}_unmapped_sorted.bam -@ 4 $OUTDIR/${prefix_name}_unmapped.bam
samtools sort -m 7G -o $OUTDIR/${prefix_name}_mapped_sorted.bam -@ 4 $OUTDIR/${prefix_name}_mapped.bam
samtools index -b -@ 4 $OUTDIR/${prefix_name}_unmapped_sorted.bam
``
